try:
    10/0
except Exception as e:
    print(f"Tenemos un error: {e}")